packages/medusa/src/api-v2/auth/middlewares.ts

```diff
@@ -17,4 +17,9 @@ export const authRoutesMiddlewares: MiddlewareRoute[] = [
     matcher: "/auth/:scope/:auth_provider",
     middlewares: [],
   },
+  {
+    method: ["DELETE"],
+    matcher: "/auth/:scope",
```

Comment:
Should this be `/auth/session`? Currently we don't support invalidating JWTs, so it's better to be more explicit (similar to how we have `/auth/session`) WDYT?

Reply:
Yeah, I think that makes sense. Will update the PR.

With respect to invalidating JWTs, I think we need to introduce a few changes now and then maybe add a more sophisticated invalidation mechanism in the future. 

On the backend, we introduce the expiry time (defaulting to 24 hours) and use that to sign JWTs upon authenticating. 

On the client (i.e. admin), we bust the token storage and no longer attach it to the request automatically.

Ofc. you would still be able to authenticate with that token until it expires, but only if you can get a hold of it, which is very, very unlikely given that we removed it from local storage.

In the future, we can think about adding JWT token invalidation on the backend too, for example, through a token blacklist or similar, but I don't think we need that right now. Let me know if you have any ideas here.

Reply:
@olivermrbl tokens with short expiry definitely reduce the attack surface and risk, so typically that should suffice.

For high-risk applications you usually have a `blacklist` of JWT tokens that were invalidated, and before every auth check you first make sure that the token wasn't invalidated. However, this beats the purpose of JWT tokens as you need to do a session check basically (or at least the same sort of operation), so I'd say we don't need to support it in the near future